neurolib/models/multimodel/builder/base/backend.py

```diff
@@ -138,10 +147,22 @@ def _replace_past_ys(self, expression, dt):
         for match in matches:
             time_past, idx, _ = match.split(",")
             idx = int(idx)
-            t_past = float(time_past.split("+")[0].strip()[7:])
-            t_past_ndx = np.around(t_past / dt).astype(int)
-            replace_with = self.PAST_Y_NUMBA.format(dt_ndt=np.abs(t_past_ndx), idx=idx)
-            expression = expression.replace(match, replace_with)
+            if time_past[0] == "t":
```

Comment:
I had to slightly change how the delayed state variables are replaced for numba `integrate` function